ciao-controller/command.go

```diff
@@ -106,12 +106,25 @@ func (c *controller) confirmTenantRaw(tenantID string) error {
 	}
 
 	if tenant == nil {
-		tenant, err = c.ds.AddTenant(tenantID)
+		_, err = c.ds.AddTenant(tenantID)
 		if err != nil {
 			return err
 		}
 	}
 
+	return nil
+}
+
+func (c *controller) confirmCNCI(tenantID string) error {
```

Comment:
With this change confirmTenant() will no longer start the CNCI. But this means:

1. The CNCI is only started when starting an instance (see startWorkload() below)
2. All the safety checks done in confirmTenant() to make sure that CNCI starting is not racey are bypassed